cxx/distributions/normal.cc

```diff
@@ -0,0 +1,111 @@
+// Copyright 2024
```

Comment:
+1 to moving impls to the cc files. I can go ahead and do this with the other files.

Reply:
Thanks!